### PR TITLE
CC-15100 Fix CI

### PR DESCRIPTION
**Pull request by kkicheglovspryker**

### CODE REVIEW

File: phpstan.neon
Comment (AsonUnique) [inline phpstan.neon]: It was announced a couple of weeks ago, that we shouldn't use this config anymore.

**Review by AsonUnique — APPROVED** [review]

**Review by michbeck — APPROVED** [review]

**Review by AsonUnique — APPROVED** [review]

**Comment by DmitryLymarenko** [comment]
@m-alama please next time do not merge anything without techlead/architect approval

**Pull request merged**
